spec/ContentDelegateSpec.js

```diff
@@ -2,7 +2,7 @@
 describe('The ContentDelegate class', function () {
   // 'tabId' values
   const tabId = 1234;
-
+  
```

Comment:
Nit, but try to be careful about this kind of tweak in a PR. I usually go line by line when doing my commits, and catch this kind of thing when I do.